azure-pipelines/build-release.yml

```diff
@@ -150,11 +150,12 @@ stages:
             inputs:
               versionSpec: '3.7'
 
-          - template: steps/install-development-dependencies.yml
-
-          - template: steps/generate-documentation.yml
+          - script: |
```

Comment:
I don't see any pipenv removal in commit "ci: Use tox instead of pipenv for docs build". Should we name the commit "ci: Use tox for docs build"?